validator/validations/steps_test.go

```diff
@@ -1,14 +1,25 @@
 package validations
 
-import "testing"
+import (
+	"testing"
+)
 
 func TestSteps_HasBeenValidated(t *testing.T) {
 	tests := []struct {
 		name string
 		s    Steps
 		want bool
 	}{
-		// TODO: Add test cases.
+		{
+			name: "test",
+			s:    255,
```

Comment:
The point of Validations is to use the bitmask. This introduces a magic value that doesn't explain what's going on. Also if we change the flags around at some point, this value is no longer correct.

So:
```go
s: validations.Steps(validations.FSyntax),  // validations.Steps is needed to coerce the type
```

This goes for all usages of `Validations` and `Steps` types in the tests

Reply:
I tried to implement:
`s: validations.Steps(validations.FSyntax),`

But then I get:  _"undeclared name: validations"_
How do I fix this?

Reply:
Oh shoot. You're in the package `validations` so you can skip that part. So just:

```go
s: Steps(FSyntax)
```